board.js

```diff
@@ -0,0 +1,47 @@
+const load = new LoadAndSaveTheBoard();
+load.valueLoad(); // loading the previous selected size of the board
+class Board {
```

Comment:
Remember each object should have some responsibility according to SOLID principle. This one represents the board of the game so all logic connected with board should be kept over here. Board should contains two dimensional array with Fields.